product_docs/docs/pgd/5/manual/08-using-pgd-cli.mdx

```diff
@@ -0,0 +1,252 @@
+ - Including the password in the connection string.
+
+We recommend the first option, as the other options don't scale well with multiple databases, or they compromise password confidentiality.
+!!!
+
+### Configuring and connecting PGD CLI
+
+* Create a YAML file which specifies the cluster and endpoints the PGD CLI application should use.
+* Install the file in the default config directory /etc/edb/pgd-cli/ as pgd-cli-config.yml.
+* Run pgd-cli.
+* Running pgd-cli without a configuration file.
+* Running pgd-cli with an alternative configuration file.
+
+### Use PGD CLI to explore the cluster
+* Check the health of the cluster with the `check-health` command.
+* Show the nodes in the cluster with the `show-nodes` command.
+* Show the proxies in the cluster with the `show-proxies` command.
+* Show the groups in the cluster with the `show-groups` command.
+* Set a group option with the `set-group-options` command.
+* Swith write leader with the `switchover` command
+
+We go into more details of these command in the worked example below.
+
+## Worked example
+
+### Creating a configuration file
+
+The PGD CLI configuration file is similar to the PGD proxy configuration filer.
+It is a YAML file which contains a cluster object. This has two properties:
+
+The name of the PGD cluster's top-level group (as `name`).
+An array of endpoints of databases (as `endpoints`).
+
+```
+cluster:
+  name: pgd
+  endpoints:
+    - host=host-one dbname=bdrdb port=5444
+    - host=host-two dbname=bdrdb port=5444
+    - host=host-three dbname=bdrdb port=5444
+```
+
+Note that the endpoints in this example specify port=5444.
+This is necessary for EDB Postgres Advanced Server instances.
+For EDB Postgres Extended and Community PostgreSQL, this can be omitted.
+
+### Installing the PGD CLI configuration
+
+Create the PGD CLI configuration directory.
+
+```
+sudo mkdir -p /etc/edb/pgd-cli
+```
+
+Then write the configuration to the `pgd-cli-config.yml` file in the `/etc/edb/pgd-cli` directory.
+
+For our example, this could be run on host-one to create the file.
+
+```
+cat <<EOF | sudo tee /etc/edb/pgd-cli/pgd-cli-config.yml
+cluster:
+  name: pgd
+  endpoints:
+    - host=host-one dbname=bdrdb port=5444
+    - host=host-two dbname=bdrdb port=5444
+    - host=host-three dbname=bdrdb port=5444
+EOF
+```
+
+### Run PGD CLI
+
+With the configuration file in place, we can run pgd-cli. For example, we can use the `show-nodes` command to list the nodes in our cluster and their status.
+
+```
+pgd show-nodes
+__OUTPUT__
+Node       Node ID    Group Type Current State Target State Status Seq ID
+----       -------    ----- ---- ------------- ------------ ------ ------
+node-one   2824718320 dc1   data ACTIVE        ACTIVE       Up     1
+node-three 1954860017 dc1   data ACTIVE        ACTIVE       Up     3
+node-two   2299992455 dc1   data ACTIVE        ACTIVE       Up     2
+```
+
+### Run PGD CLI without a configuration file
```

Comment:
Isn't this already covered here:  https://deploy-preview-5104--edb-docs-staging.netlify.app/docs/pgd/latest/cli/using_cli/#passing-a-database-connection-string

Reply:
Pulled for a reference to the main docs.